src/data_access/models/serializable_model.py

```diff
@@ -0,0 +1,8 @@
+from src.data_access.database import db
+
+
+class SerializableModel(db.Model):
+    __abstract__ = True
+
+    def to_dict(self):
```

Comment:
dont have to fix but the names serialize and deserialize are used.. and add a return type :)